src/coreclr/tests/issues.targets

```diff
@@ -2719,4 +2719,9 @@
         </ExcludeList>
         <!-- End interpreter issues -->
     </ItemGroup>
```

Comment:
Just confirming, the original file is based off RC1/2 or off main ?  (In main this file issues.target, has been moved to https://github.com/dotnet/runtime/tree/master/src/tests )

Reply:
its based off rc2